packages/ui/ant-design/src/ConnectionModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useSetConnection } from '@solana/wallet-adapter-react';
+import { Menu, Modal, ModalProps } from 'antd';
+import React, { FC, MouseEvent, useCallback, useMemo, useState } from 'react';
+import { useConnectionModal } from './useConnectionModal';
+import { ConnectionMenuItem } from './ConnectionMenuItem';
+import { Connection } from '@solana/web3.js';
+
+export interface ConnectionModalProps extends Omit<ModalProps, 'visible'> {
+    connections: {
+        name: string;
+        endpoint: string;
+    }[];
```

Comment:
Always good to export named interfaces for nested props if they need to be consumed by descendants and downstream devs.